src/components/Dropdown/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from "react";
+
+export default function Dropdown(props) {
+	const [isExpanded, setIsExpanded] = useState(true);
+  const {label, dropdown, submenu} = props;
+	function dropMenu () {
+		setIsExpanded(!isExpanded);
+	}
+	const dropClass = isExpanded ? "dropdown__content-off" : "dropdown__content";
+  return (
+  	<div className="dropdown">
+      <label>{label}</label>
+  		<button className="dropdown__btn" onClick={ dropMenu }>
+        {dropdown}
+  		</button>
+  		<div className= { dropClass }>
+  			<ul>
+  			 <li><a href="#">{submenu}</a></li>
```

Comment:
No, no, no you should send over array of items in the props, for example:

`[{ label: "item-1", value: "value-1"}, { label: "item-2", value: "value-2"}]`

and then you should use 

```
_.map(items, item => (
<li><a href="#">{item.label}</a></li>
))
```

also, you'll need to figure out, how to return somehow selected item with `onClick`, right?